install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -138,6 +138,12 @@ data:
   # for cilium-health.
   agent-health-port: "{{ .Values.healthPort }}"
 {{- end }}
+
+{{- if hasKey .Values "clusterHealthPort" }}
```

Comment:
Just came to my mind after I submitted my review: Do we also want to add this to the `values.yaml` so it's part of the generated docs? https://github.com/cilium/cilium/blob/master/Documentation/helm-values.rst

Reply:
As long as we set the value to the same as the current default, I think this should be fine from an upgrade perspective.

Reply:
I thought this would be okay as a hidden value, the other port parameter I'm trying to add is more complicated (see https://github.com/cilium/cilium/pull/16874#discussion_r669556969), so I though having both hidden would make sense.